lib/plugin.js

```diff
@@ -181,10 +181,12 @@ const setupProgress = (axios) => {
 }<% } %>
 
 export default (ctx, inject) => {
+  // runtimeConfig
+  const runtimeConfig = ctx.$config && ctx.$config.axios || {}
   // baseURL
   const baseURL = process.browser
-      ? '<%= options.browserBaseURL || '' %>'
-      : (process.env._AXIOS_BASE_URL_ || '<%= options.baseURL || '' %>')
+    ? (runtimeConfig.browserBaseURL || runtimeConfig.baseURL || '<%= options.browserBaseURL || '' %>')
+      : (process.env._AXIOS_BASE_URL_ || runtimeConfig.baseURL || '<%= options.baseURL || '' %>')
```

Comment:
Maybe we can use runtimeConfig with more periority than `_Axios_BASE_URL_` to keep consistency with client?

Reply:
I didn't noticed this comment before 🤦